cli/types/cypress.d.ts

```diff
@@ -786,14 +786,20 @@ declare namespace Cypress {
      */
     off: Actions
 
+    /**
+     * Used to include dependencies within the cy.origin() callback
+     * @see https://on.cypress.io/origin
+     */
+    require: (id: string) => any
```

Comment:
I wonder if we can leverage generics here to allow users to have a quick escape hatch to add types if using Typescript? That way they can just do something like:

```ts
cy.origin('<origin>', () => {
  const lodash = Cypress.require<LoDashStatic>('lodash')
})
```
```suggestion
    require: <T>(id: string) => T
```

if they don't provide the type, the type is just `unknown` which seems fine.

Reply:
Good idea. I'll update the docs with this too. I defaulted to `any` since otherwise, at least for our own TS configuration, it errors when trying to use methods on an `unknown` type.

Addressed in https://github.com/cypress-io/cypress/pull/25626/commits/b5270d5fad626668014d648d7199674cac7e3d0c